Makefile.j2

```diff
@@ -60,19 +60,37 @@ run-semsim: generate-ontologies
 
 run-semsim: {{ directories.profiles }}/{{ semsim.ontology }}-{{ semsim.subset }}.{{semsim.threshold}}.{{ semsim.method }}.tsv
 
-{{ directories.profiles }}/{{ semsim.ontology }}-{{ semsim.subset }}.{{semsim.threshold}}.{{ semsim.method }}.tsv: $(ONTOLOGYDIR)/{{ semsim.ontology }}.db $(TMP_DATA)/hp_terms.txt $(TMP_DATA)/mp_terms.txt $(TMP_DATA)/zp_terms.txt
+
+
+{% if 'ic' in semsim -%}
```

Comment:
You can merge this, but remember to separate this stuff into a different jinja template eventually.